src/App.tsx

```diff
@@ -1,5 +1,41 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+import { AuthContext } from './context/AuthProvider';
+import ProtectedRoute from './ProtectedRoute';
 
-const App: React.FC = () => <h1>hello world</h1>;
+const App: React.FC = () => {
+  const user = useContext(AuthContext);
+
+  return (
+    <div>
+      <Router>
```

Comment:
Routes coukd be stored in the separate folder like `Routes` where you could move `ProtectedRoute` and create new file for below routes (try to create more files for connected logic)